app/src/main/java/nl/tudelft/bbw/blockchain/Acquaintance.java

```diff
@@ -15,14 +16,14 @@
     /**
      * Properties of a Acquaintance
      */
-    private ArrayList<Chain> multichain;
+    private List<List<Block>> multichain;
```

Comment:
The chain class is still present in the UML. Can you please update the UML then?